packages/detox/src/migrations/update-13-8-2/remove-types-detox-13-8-2.ts

```diff
@@ -0,0 +1,26 @@
+import { getProjects, readJson, Tree, updateJson } from '@nrwl/devkit';
+
+export default async function update(tree: Tree) {
+  const packageJson = readJson(tree, 'package.json');
+
+  if (!packageJson.devDependencies['@types/detox']) {
+    return;
+  }
+
+  const projects = getProjects(tree);
+  let hasDetoxProject = false;
```

Comment:
```
cons hasDetoxProject = projects.some(project => {
  return project.targets?.['test-ios']?.executor === '@nrwl/detox:test';
});
```
More clear, and breaks early if project is found.

Reply:
the `projects` is a Map type, which is not array, so it does not have .some function.